RELEASE/scripts/autoscend/auto_island_war.ash

```diff
@@ -271,38 +317,34 @@ boolean L12_startWar()
 		use_skill(1, $skill[Incredible Self-Esteem]);
 	}
 
-	autoAdv(1, $location[Wartime Hippy Camp]);
-	set_property("choiceAdventure142", "3");
-	if(contains_text(get_property("lastEncounter"), "Blockin\' Out the Scenery"))
+	// set noncombats to value needed to start the war
+	set_property("choiceAdventure142", "3");	//if wearing [Frat Warrior Fatigues] start the war or skip adventure
+	set_property("choiceAdventure146", "3");	//if wearing [War Hippy Fatigues] start the war or skip adventure
+
+	// wear the appropriate war outfit based on auto_hippyInstead
+	warOutfit(false);
+
+	// start the war when siding with frat boys
+	if(!get_property("auto_hippyInstead").to_boolean())
 	{
-		if(!get_property("auto_hippyInstead").to_boolean())
+		auto_log_info("Must save the ferret!!", "blue");
+		autoAdv(1, $location[Wartime Hippy Camp]);
+		
+		if(contains_text(get_property("lastEncounter"), "Blockin\' Out the Scenery"))
 		{
-			visit_url("bigisland.php?action=junkman&pwd");
-			visit_url("bigisland.php?place=concert&pwd");
-			visit_url("bigisland.php?place=lighthouse&action=pyro&pwd=");
+			visit_url("bigisland.php?place=concert&pwd");					//immediately acquire flyers for frat
 			visit_url("bigisland.php?place=lighthouse&action=pyro&pwd=");
+			visit_url("bigisland.php?place=lighthouse&action=pyro&pwd=");	//immediately turn in lighthouse quest

```

Comment:
Is there any point to this? I assume it's because you could in theory have faxed/wished a Lobsterfrogman and copied it 4 times before you even started the war so you have the barrels but that should get handled in `L12_sonofaFinish()` regardless.

Reply:
You are right. this is pointless, and it is exactly for the reason you stated, and it is already handled by L12_sonofaFinish()

change made